src/Units/UnitModel.java

```diff
@@ -15,8 +15,16 @@
     INFANTRY, MECH, RECON, TANK, MD_TANK, NEOTANK, APC, ARTILLERY, ROCKETS, ANTI_AIR, MOBILESAM, FIGHTER, BOMBER, B_COPTER, T_COPTER, BATTLESHIP, CRUISER, LANDER, SUB
   };
 
+  // NB: "Truck" is technically unnecessary, but I feel it may be useful for alternate damage systems.
+  // The rubric for what's a "truck" is whether the damage dealt to it by an infantry is >9%, not movetype.
```

Comment:
Oh. I see. I don't think I like infantry damage as the distinguishing factor, especially since, as you say, this could be used for other damage systems (in which infantry may not deal >9% damage to certain of these vehicles). I might be feeling "domain" as a more useful discriminator than "chassis".